10_simplification/main.go

```diff
@@ -145,31 +147,122 @@ can be taken.
+			fmt.Println(k)
+		}
+	}
 	// TODO: write a loop that prints the first 10 odd numbers
+	for l := 0; l < 10; l++ {
+		if l%2 != 0 {
+			fmt.Println(l)
+		}
+	}
 	// TODO: write a loop that calculates the sum of the first 10 numbers
+	var sumFirstTen int
+	for m := 0; m < 10; m++ {
+		sumFirstTen += m
+	}
+	// debug
+	fmt.Println(sumFirstTen)
 	// TODO: write a loop that calculates the sum of the first 10 odd numbers
+	var sumFirstTenOdd int
+	for n := 0; n < 10; n++ {
+		if n%2 != 0 {
+			sumFirstTenOdd += n
+		}
+	}
+	// debug
+	fmt.Println(sumFirstTenOdd)
+
 	// TODO: write a loop that prints the first 10 numbers, except 5 and 7
+	for p := 0; p < 10; p++ {
+		if p == 5 {
+			continue
+		}
+		if p == 7 {
+			continue
+		}
+		fmt.Println(p)
+	}
+
 	// TODO: write a loop that sums numbers until it reaches 20, print each new
 	// sum
 	// e.g. 0, 0+1, 0+1+2, 0+1+2+3, ...
-
-	// TODO: write a function that sums 3 `int` and returns the result
-	// TODO: write a function that multiplies 3 `int` and returns the result
-	// TODO: write a function that returns the sum of all numbers between its
-	// parameters.
-	// If the second parameter is less than the first, return 0.
-	// e.g. sum(1, 3) => 1 + 2 + 3 = 6
-	//      sum(2, 5) => 2 + 3 + 4 + 5 = 14
+	var sumTwenty int
+	for r := 0; r < 20; r++ {
+		sumTwenty += r
+		fmt.Println("Print sumTwenty", sumTwenty)
+	}
 
 	// TODO: declare an array of 5 integers and print it
+	arrayFiveInt := []int{1, 2, 4, 5, 6}
+	fmt.Println("Print arrayFiveInt", arrayFiveInt)
 	// TODO: declare an array of 5 strings and print it
+	arrayFiveStrings := []string{"ISO 8601", "UTC", "FOR", "GLOBAL", "STANDARDS"}
+	fmt.Println("Print arrayFiveStrings", arrayFiveStrings)
 	// TODO: declare a map with 3 key-value pairs and print it
+	mapThreePairs := map[string]string{
+		"area":       "Seoul Metro region",
+		"population": "~25 million",
+		"language":   "Korean",
+	}
+	fmt.Println("Print mapThreePairs", mapThreePairs)
+
+	// close main function
+	fmt.Println("Print sumThreeInt", sumThreeInt(1, 2, 3))
+
+	fmt.Println("Print mulThreeInt", mulThreeInt(1, 2, 3))
+
+	fmt.Println("Print returnBetween", returnBetween(1, 2))
+	fmt.Println("Print returnBetween", returnBetween(1, 9))
+	fmt.Println("Print returnBetween", returnBetween(-1, 2))
+	fmt.Println("Print returnBetween", returnBetween(0, 0))
+	fmt.Println("Print returnBetween", returnBetween(3, 5))
+
+}
+
+// TODO: write a function that sums 3 `int` and returns the result
+func sumThreeInt(a, b, c int) int {
+	return a + b + c
+}
+
+// TODO: write a function that multiplies 3 `int` and returns the result
+func mulThreeInt(a, b, c int) int {
+	return a * b * c
+}
+
+// TODO: write a function that returns the sum of all numbers between its
+// parameters.
+// If the second parameter is less than the first, return 0.
+// e.g. sum(1, 3) => 1 + 2 + 3 = 6
+//
+//	sum(2, 5) => 2 + 3 + 4 + 5 = 14
+func returnBetween(a, b float64) float64 {
+	var sum float64
+	if b < a {
+		return 0
+	}
```

Comment:
You only need `sum` if you use it, so you can define it later

```suggestion
	if b < a {
		return 0
	}
	
	var sum float64
```

Reply:
Done!